docs/types/entity_action_types/modify_inventory.md

```diff
@@ -0,0 +1,45 @@
+---
+title: Modify Inventory (Entity Action Type)
+date: 2022-6-07
+---
+
+# Modify Inventory
+
+[Entity Action Type](../entity_action_types.md)
+
+Selects items from anywhere in a player's inventory to modify them with an [Item Action](../types/item_action_types.md).
+
+Type ID: `origins:modify_inventory`
+
+### Fields
+
+Field  | Type | Default | Description
+-------|------|---------|-------------
+`inventory_type` | [Identifier](../../data_types/identifier.md) | `inventory` | Takes one of either `inventory` or `power`. Determines whether you're modified items from the player inventory, or the extra inventory from the [`origins:inventory`](../types/power_types/inventory.md) power type respectively.
+`entity_action` | [Entity Action](../types/entity_action_types.md) | _optional_ | If specified, this action will be ran after the items are modified.
```

Comment:
```suggestion
`entity_action` | [Entity Action](../types/entity_action_types.md) | _optional_ | If specified, this action will be executed on the entity before the items are modified.
```
In general, the entity action types in the `*_inventory` entity action types are executed *before* the other action types